tests/external_processing/test_apm.py

```diff
@@ -0,0 +1,79 @@
+from tests import test_scrubbing as base_scrubbing
+from tests import test_config_consistency as base_config_consistency
+from tests import test_standard_tags as base_standard_tags
+from tests import test_semantic_conventions as base_semantic_conventions
+from utils import weblog, interfaces, scenarios, features
+
+
+@features.not_reported  # it's just a POC. We'll need to figure out how we want to see results in FPD
```

Comment:
Can you create a feature in the FPD? (ping me if you need help)

Reply:
Applied in 6a4a9ad 😄